CONTRIBUTING.md

```diff
@@ -25,8 +25,8 @@ There are quite a few ways to contribute to the project, find inspiration from t
 
 |Activity|Description|
 |-|-|
-|Community Q&A|We encourage you to ask and answer technical questions on [GitHub discussions](https://github.com/kedro-org/kedro/discussions) or [Discord](https://discord.gg/akJDeVaxnB), but the former is often preferable since it will be picked up by search engines.|
-|Report a bug|We use [GitHub issues](https://github.com/kedro-org/kedro/issues) to keep track of known bugs. We keep a close eye on them and update them when we have an internal fix in progress. Before you report a new issue, do your best to ensure your problem hasn't already been reported. If it has, just leave a comment on the existing issue, rather than create a new one. <br /> If you have already checked the existing [GitHub issues](https://github.com/kedro-org/kedro/issues) and are still convinced that you have found odd or erroneous behaviour then please file a new one.|
+|Community Q&A|We encourage you to ask and answer technical questions on [GitHub discussions](https://github.com/kedro-org/kedro/discussions) or [Slack](https://slack.kedro.org/), but the former is often preferable since it will be picked up by search engines.|
```

Comment:
I hadn't seen this comment previously! I'd like to remove the Q&A from GH discussions so we keep it all on Slack going forwards since it's confusing and duplicates effort if we spread it across channels. I've commented to this effect [here](https://github.com/kedro-org/kedro/discussions/1986) and [here](https://github.com/kedro-org/kedro/discussions/1987) on GH discussions and have discussed with Merel if we can make the Q&A section read-only somehow.

Also, I'm not sure now whether the Slack channel will be indexed by search engines?

Reply:
So maybe just say "We encourage you to ask and answer technical questions on [Slack](https://slack.kedro.org/)."